core/events/events.go

```diff
@@ -15,6 +15,10 @@ const (
 	FileBackupInProgress FileEventType = "FileBackupInProgress"
 	FileBackupReady      FileEventType = "FileBackupReady"
 
+	FileRestoreInProgress FileEventType = "FileRestoreInProgress"
+	FileRestored          FileEventType = "FileRestored"
+	FileRestoring         FileEventType = "FileRestoring"
```

Comment:
What's the difference between `FileRestoring` and `FileRestoreInProgress`? Maybe we can call the formed `FileRestoreStarted`?

Reply:
@jsonsivar 

Thank you for spotting this, a left-over from reshuffles, fixed in d5279b68c5ed743ec1796130e8c29345266097d0